packages/svelte/src/reactivity/map.js

```diff
@@ -155,4 +156,14 @@ export class ReactiveMap extends Map {
 	get size() {
 		return get(this.#size);
 	}
+
+	[INSPECT_SYMBOL]() {
```

Comment:
We should probably add these to the prototype only in DEV to reduce bundle size.